dbt_project.yml

```diff
@@ -50,7 +50,7 @@ seeds:
 models:
   spellbook:
     +pre-hook:
-      - sql: "{{ set_trino_session_property(is_materialized(model) and is_partitioned(model), 'join_distribution_type', 'PARTITIONED') }}"
+      - sql: "{{ set_trino_session_property(is_materialized(model) and is_partitioned_enforced(model), 'join_distribution_type', 'PARTITIONED') }}"
```

Comment:
Nit: Could we make this more flexible and allow for configuring any join distribution type on the model. E.g.,
```
{{ set_trino_session_property(model.config.get('join_distribution_type', false), 'join_distribution_type', model.config.get('join_distribution_type'))) }}
```

Reply:
Or add a new macro to support a simpler syntax.

Reply:
Yeah - this makes more sense 👍 I've opened this property to all of the property's possible values.